packages/fetchai/protocols/consensus/README.md

```diff
@@ -0,0 +1,42 @@
+# Consensus Protocol
+
+## Description
+
+This is a consensus protocol for aggregating observations.
+
+## Specification
+
+```yaml
+---
+name: consensus
+author: fetchai
+version: 0.1.0
+description: A protocol for agents to aggregate individual observations
+license: Apache-2.0
+aea_version: '>=0.9.0, <0.10.0'
+speech_acts:
+  observation:
+    value: pt:int
+    time: pt:int
+    source: pt:str
+    signature: pt:str
+  aggregation:
+    value: pt:int
+    time: pt:int
+    contributors: pt:list[pt:int]
+    signature: pt:str
+...
+---
+initiation: [observation, aggregation]
```

Comment:
Is it just two independent messages?

Reply:
Yes, I hope to add more logic here in the future, but for now the protocol is very light and these two message types can be sent independently.